src/db/migrations/20190717164759-create-tag.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Tags', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
```

Comment:
Hi @devPinheiro, is there a reason for not validating the name not to allow null cases?

Reply:
Yes, there is a reason for this. The feature is implemented such that users can choose not to add tags to their article. Users can also choose to have any description for their tag.